src/measure_theory/measure/sub.lean

```diff
@@ -72,9 +72,9 @@ begin
         tsub_add_cancel_of_le (h₂ t h_t_measurable_set)] },
     have h_measure_sub_eq : (μ - ν) = measure_sub,
     { rw measure_theory.measure.sub_def, apply le_antisymm,
-      { apply @Inf_le (measure α) measure.complete_semilattice_Inf,
+      { apply @Inf_le (measure α) measure.complete_lattice,
```

Comment:
Do we really need `@` here? What if you use `refine` instead of `apply`?

Reply:
`refine` works.